images/router/haproxy/conf/error_code_pages/error-page-503.http

```diff
@@ -0,0 +1,140 @@
+HTTP/1.0 503 Service Unavailable

```

Comment:
If we are moving the 503 page to `images/router/haproxy/conf/error_code_pages/error-page-503.http`, we can delete `images/router/haproxy/conf/error-page-503.http`, right?

Reply:
Yeah I had copied both files from the existing ones. So images/router/haproxy/conf/error_code_pages/ is for custom error pages and images/router/haproxy/conf/ for original pages. If custom pages is not provided then original pages will be served.

Reply:
If custom error pages are user provided than why do we need to copy the original error pages?

Reply:
If custom error pages are user provided than why do we need to copy the original error pages?